components/gitpod/gitpod.libsonnet

```
@@ -951,6 +951,32 @@ function(params) {
     },
   },
 
+  workspacePodMonitor: {
+    apiVersion: 'monitoring.coreos.com/v1',
+    kind: 'PodMonitor',
+    metadata: {
+      name: 'workspace',
+      namespace: $._config.namespace,
+      labels: $._config.commonLabels,
+    },
+    spec: {
+      selector: {
+        matchLabels: {
+          'component': 'workspace',
+        },
+      },
+      namespaceSelector: {
+        matchNames: [
+          $._config.gitpodNamespace,
+        ],
+      },
+      podMetricsEndpoints: [{
+        port: 'supervisor',
+        interval: '30s',
+      }],
```

I believe something like this would do the trick, but would be nice to try it out on a preview environment just to be sure.

```suggestion
      podMetricsEndpoints: [{
        port: 'supervisor',
        interval: '30s',
        relabelings: [{
          source_labels: '[__name__]',
          regex: 'gitpod_*',
          action: 'keep',
        }]
      }],
```